.travis.yml

```diff
@@ -51,16 +51,21 @@ matrix:
     rust: stable
     osx_image: xcode10.2
 
-addons:
-  apt:
-    update: true
-    sources:
-      - llvm-toolchain-xenial-7
-    packages:
-      - clang-tidy-7
-      - llvm-7-dev
-      - libclang-7-dev
-      - clang-7
+  - os: linux
+    arch: arm64
+    rust: stable
+    install:
+      - if ! which go 2>/dev/null; then
+          wget https://dl.google.com/go/go1.13.8.linux-arm64.tar.gz;
+          tar -xf go1.13.8.linux-arm64.tar.gz -C $HOME;
+          export PATH="$HOME/go/bin:$PATH";
+          export GO_ROOT=$HOME/go;
+        fi
+    addons:
+      apt:
+        update: true
+        packages:
```

Comment:
generating bindgen needs LLVM support

Reply:
yes. It seems the default CI environment already install llvm tool chains. On the other hand, there are no llvm apt repo available on aarch64/linux, thus made it hard to install llvm tool chain on this platform, so just use the system default packages maybe a good idea currently.